Cargo.toml

```diff
@@ -131,6 +131,7 @@ cli = [
 aleo-cli = [ "snarkvm-synthesizer/aleo-cli" ]
 async = [ "snarkvm-ledger/async", "snarkvm-synthesizer/async" ]
 cuda = [ "snarkvm-algorithms/cuda" ]
+save_r1cs_hashes = [ "snarkvm-circuit/save_r1cs_hashes" ]
```

Comment:
We seem to have two conventions for feature flags (dashes or snake_case). Not something for this PR but we should consider aligning these in future. IIRC snarkOS mainly uses the former.